gapis/service/path/path.proto

```diff
@@ -369,6 +369,8 @@ message Slice {
 // State is a path to a subset of the GlobalState at a point in a capture.
 message State {
     Command after = 1;
+    // If non-nil, then the state is filtered to the specified context.
+    ID context = 2;
```

Comment:
Won't this break links to objects in another context which is shared?
```
ctx1 = newContext(nil)
ctx1...[1] = newTexture()
ctx2 = newContext(ctx1)
ctx2.bindTexture(1)
```

If in the UI I am filtering to ctx2, the link will attempt to go to ctx1, which is now no longer part of the tree, right?

Reply:
No it won't - because the server will return absolute GlobalState paths for followed links. The state tree nodes will also use GlobalState paths so that everything is absolute.

Reply:
State paths are used as relative paths to the most relevant part of the GlobalState (as considered by GAPIS).